annoying/tests/fields.py

```diff
@@ -1,8 +1,42 @@
+import json
+
 from django.test import TestCase
 from . import models
 
 
+def dump_dict(d):
+    """
+    Returns a JSON string sorted by keys
+    """
+    return json.dumps(d, sort_keys=True)
+
+
 class FieldsTestCase(TestCase):
     def test_auto_one_to_one(self):
         super_villain = models.SuperVillain.objects.create()
         self.assertEqual(super_villain.mortal_enemy.name, "Captain Hammer")
+
+    def test_json_field_create(self):
+        stats = {
+            'strength': 100,
+            'defence': 50,
+        }
+        super_villain = models.SuperVillain.objects.create(stats=stats)
+        self.assertEqual(super_villain.stats['strength'], 100)
```

Comment:
You're getting a new instance below, in test_update, so that's probably tested enough. You may want to test the `.create()`, in which case it would be good to get it from the DB again, but I'm most worried about people being confused by thinking at a cursory glance that this test retrieves the object from the db when it doesn't. Of the two options, I'd go with getting it from the DB again.